apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V59__fiss_claims_prov_fields.sql

```diff
@@ -0,0 +1,4 @@
+alter table "pre_adj"."FissClaims" add "provStateCd" varchar(2);
+alter table "pre_adj"."FissClaims" add "provTypFacilCd" varchar(1);
+alter table "pre_adj"."FissClaims" add "provEmerInd" varchar(1);
+alter table "pre_adj"."FissClaims" add "provDeptId" varchar(3);
```

Comment:
@keithdadkins when you have a chance can you verify that we have the correct ownership on this table in all envs?